import.py

```diff
@@ -651,9 +652,8 @@ def get_decompme_compiler_name(
 
     for path, compiler_name in compiler_mappings.items():
         assert isinstance(compiler_name, str)
-        # This is an annoying hack because decompme doesnt seem to know about macos
-        compiler_path = compiler_path.replace("macos", "linux")
-        if path == compiler_path:
+
+        if fnmatch.fnmatch(compiler_path, path):
```

Comment:
Oh, convenient! Can you extend the error messages below with " (wildcards allowed)" or something like that?

Reply:
Done!